TODO.txt

```diff
@@ -1,47 +0,0 @@
-- affine group of an origami
```

Comment:
@videlec is this file still relevant? I guess all these should be issues instead?

Reply:
This is now https://github.com/flatsurf/surface-dynamics/issues/104.